proposal-selection.md

```diff
@@ -0,0 +1,108 @@
+
+**Test the waters:**
+
+*   Float an idea in our community forums.
+*   Look for feedback, interest and engagement. 
+*   If you attract significant community interest and support, and the required effort is more than is likely to be addressed by volunteers, you might consider developing a proposal for sponsored work.
+
+**Prepare proposal:**
+
+*   Develop a proposal in conjunction with the community. Make sure it addresses our project priorities.
+*   Find a trusted mentor from our community who is willing to mentor you.
+*   Share proposal.
+
+**Review proposals:**
+
+*   Community review and discuss proposals.
+*   Community decides if the proposal is worth backing, and locks in if it is.
+
+**Implementation phase:**
+
+*   The worker works according to the proposed plan.
+*   The worker checks in weekly with mentors and other collaborators.
+*   Worker provides a weekly status. This can be brief, just a few lines.
+
+**Wrap up:**
+
+*   Write a report, hand over documents and present to the community.
+*   Send an announcement to the main mailing and slack #announcements channel.
+*   Arrange for a tweet and/or blog post to be created announcing the project completion, linking to the finished work.
+
+## Payment process
+
+*   Where applicable, guidance on budget will typically be based on the [stipend amounts](https://developers.google.com/season-of-docs/docs/tech-writer-stipends) used by Season of Docs (20-30 hours per week x 3 months, or equivalent effort spread over 6 months). This will be referenced from the Request for Proposal.
+*   Payment milestones will typically be every 3 months.
+*   Payments should be correlated to milestones as github tasks. 
+*   When a milestone is reached:
+    *   The implementor should comment on the milestone task, asking the mentor for review. 
+    *   The implementor should raise an invoice with our [Open Collective account](https://opencollective.com/thegooddocsproject/expenses/new).
+    *   The mentor should review the milestone, and either approve it (by closing the issue) or reject it (by providing feedback on what is still needed to reach the milestone).
+    *   If the milestone is approved, the mentor should raise a motion with the  PSC recommending that the payment be approved.
+    *   Once approved, a treasurer (not the mentor) should approve the Open Collective invoice, and the mentor updates the decision log with payment approval.
+
+## Project Adjustments
+
+*   Project scope may be adjusted by mutual agreement between the mentor and proposer. 
+*   Scope changes must be approved by the mentor and PSC [decision process](decision.html).
+*   All scope changes must be documented in writing.
+
+## Project completion
+
+*   Projects are not considered complete until all pull requests associated with the scope of work are merged and any final project report (if part of the original project proposal) has been submitted.
```

Comment:
My one thought here about this process -- often times work is hard to scope up front, and timelines and deliverables can be hard to nail down in the beginning.  My concern here is we could end up quoting budgets/spending based on a project that isn't easily defined and what was going to take a month takes 3 months.  This results in issues around people wanting paid for work that isn't fully complete due to the `unknowns` of the work.  Might be worth figuring out a process in which we can allocate payments and budgeting around an agile-workflow... happy to discuss more on this.